server/src/main/java/org/elasticsearch/Version.java

```diff
@@ -178,7 +178,8 @@ public class Version implements Comparable<Version>, ToXContentFragment {
     public static final Version V_7_17_17 = new Version(7_17_17_99, org.apache.lucene.util.Version.LUCENE_8_11_1);
     public static final Version V_7_17_18 = new Version(7_17_18_99, org.apache.lucene.util.Version.LUCENE_8_11_1);
     public static final Version V_7_17_19 = new Version(7_17_19_99, org.apache.lucene.util.Version.LUCENE_8_11_1);
-    public static final Version CURRENT = V_7_17_19;
+    public static final Version V_7_17_20 = new Version(7_17_20_99, org.apache.lucene.util.Version.LUCENE_8_11_3);
+    public static final Version CURRENT = V_7_17_20;
```

Comment:
I am getting a bunch of failures in CI about my version stuff. 

Should I keep `V_7_17_19` and just change the Lucene version since its a bugfix version and fully compatible?

@elastic/es-core-infra

Reply:
Yes, new versions are only introduced at build release time. Just update the 7.17.19 constant